test/valgrind/valgrindTestSuite.sh

```diff
@@ -456,6 +472,25 @@ function failedTest()
 }
 
 
+
+# -----------------------------------------------------------------------------
+#
+# memErrorFound
+#
+function memErrorFound()
+{
+  _valgrindFile=$1
+  _file=$2
+  _memErrors=$3
+
+  echo "FAILED (mem errors: $_memErrors). Check $_valgrindFile for clues"
```

Comment:
Both `leakInfo` and `memErrorInfo` are called, so we will see both error types if present